istioctl/pkg/describe/describe.go

```diff
@@ -220,19 +221,19 @@ func extendFQDN(host string) string {
 }
 
 // getDestRuleSubsets gets names of subsets that match any pod labels (also, ones that don't match).
-func getDestRuleSubsets(subsets []*v1alpha3.Subset, podsLabels []klabels.Set) ([]string, []string) {
-	matchingSubsets := make([]string, 0, len(subsets))
-	nonmatchingSubsets := make([]string, 0, len(subsets))
+func getDestRuleSubsets(subsets []*v1alpha3.Subset, podsLabels []klabels.Set) (sets.String, sets.String) {
```

Comment:
Why is this changing? It seems worse, now we lose ordering

Reply:
+1